boards/arm/96b_carbon/96b_carbon.dts

```diff
@@ -16,6 +16,37 @@
 		zephyr,sram = &sram0;
 		zephyr,flash = &flash0;
 	};
+
+	leds {
+		compatible = "gpio-leds";
+		green_led_1: led@0 {
```

Comment:
Since we provide a dedicated device node for LED shouldn't we also provide a dedicated device driver, i.e. LED API? The current approach is somewhat inconsistent.

Reply:
Proposed LED objects could be driven with gpio driver.
Proposed compatible/yaml allow to generate information that could be used with GPIO driver.
I'm not sure a LED driver is required for current purpose. But I expect an upcoming LED driver could be based on this.
In the opposite case, a specific compatible could be defined with the new driver.